packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -89,9 +97,10 @@ contract Attestations is
   struct Attestation {
     AttestationStatus status;
 
-    // For outstanding attestations, this is the timestamp of the request
-    // For completed attestations, this is the timestamp of the attestation completion
-    uint128 time;
+    // the timestamp of the request
+    uint64 requestTime;
+    // the timestamp of the attestation completion
+    uint64 completionTime;
```

Comment:
Do we need to store this?

Reply:
We used to store both in a single variable `time` so it was "free" already. It is important to highlight the time of the attestations imo, but it would be somewhat implicit in the `requestTime`, so we probably do not actually need `completionTime`. I actually just realized that we will need to save the block number instead of the time for the request anyways